application/templates/datamanager/assign-entities-check-results.html

```
@@ -141,6 +156,14 @@
                     <div class="app-duplicate-new-entity">{{ row.new_organisation or row.new_organisation_entity or '' }}</div>
                     <div class="app-duplicate-old-entity">{{ row.old_organisation or row.old_organisation_entity or '' }}</div>
                   </td>
+                  {% for col in duplicate_columns %}
+                  <td class="govuk-table__cell app-wrap">
+                    {% set new_value = row.new_fields.get(col, '') %}
+                    {% set old_value = row.old_fields.get(col, '') %}
+                    <div class="app-duplicate-new-entity">{% if new_value | length > 50 %}{{ new_value[:50] }}...{% else %}{{ new_value }}{% endif %}</div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)